sql_3/HomeWork_Sql_3.sql

```diff
@@ -0,0 +1,104 @@
+('et','Qui iste repudiandae.','1999-01-22','quas','77364451591856','324','1308.00','75','Simeon','Hayes','xgerlach@example.com','93','2018-09-11','quia'),
+('aut','Nesciunt ut non.','1979-04-30','assumenda','79429380293004','263','1177.76','91','Theodora','Lindgren','connelly.dakota@example.com','20','1975-03-22','ab'),
+('saepe','Quibusdam laboriosam possimus.','1998-10-13','quod','87564333151094','695','377.80','80','Audie','Mertz','citlalli.jacobs@example.com','110','2012-05-27','minima'),
+('quibusdam','Consectetur nostrum et.','1971-05-24','corporis','90837096488103','226','721.21','18','Autumn','Moen','marilie97@example.com','116','1987-02-21','ut'),
+('nemo','Non nesciunt.','2017-10-30','fugit','96684111561626','995','665.34','72','Giovanna','Lehner','brown38@example.org','58','1987-11-05','cumque'),
+('placeat','Vero eveniet.','1986-01-05','consequuntur','97858234937302','685','960.80','57','Adela','Purdy','mlangosh@example.org','21','1978-07-25','exercitationem'); 
+
+# витягнути всі книжки
+select * from book3;
+
+# обновити прізвище автора в 2х книжках
+update book3 set s_name = "Default" where id in (1,5);
+
+# підрахувати кількість книжок, в яких автори мають нагороди
+select count(*) from book3 where reward != 'NULL';
+
+# вивести інформацію про книжку з найменшою кількістю сторінок
+select * from book3 where count_pages = ( select min(count_pages) from book3 );
+
+# вивести максимальну кількість сторінок
+select max(count_pages) from book3;
+
+# вивести середню ціну між всіма книжками
+select avg(count_pages) from book3;
+
+# порахувати суму книжок за категорією
+select category, count(category) as 'сума за категорією' from book3 group by category;
+
+# вивести мінімальну ціну книжки
+select min(price) as 'min price' from book3;
+
+# вивести інформацію про книжку з найменшою ціною
+select * from book3 where price = ( select min(price) from book3 );
+
+# вивести книжки, де ціна матиме 4 знаки після коми, відсортовано за назвою книги
+select id, title, description, release_date, category, count_pages, count_of_books, round(price, 4) as 'price' from book3 order by title;
+
+# вивести книжки за limit (5, 5), (10, 6) та відсортувати за датою публікації.
+select * from book3 order by release_date limit 5, 5;
+select * from book3 order by release_date limit 10, 6;
+
+# написати 3 запити, використавши різні види пошуку за співпадіннями(like)
+select * from book3 where title like 'e%';
+select * from book3 where category like '_ui%';
+select * from book3 where release_date like '%-05-%';
+
+# вивести книжки де ціна на проміжку: 1. 50-120, 2. не включаючи проміжок 200-600.
+select * from book3 where price between 50 and 120;
+select * from book3 where price not between 200 and 600 order by price;
+
+# 3 запити на видалення книжки(за id, name, isbn)
+set sql_safe_updates = 0;
+delete from book3 where id = 25; 
+delete from book3 where price > 1400; 
+delete from book3 where title = 'nemo';
```

Comment:
Without it, I can't run these lines

Reply:
Yes, so you could put `set sql_safe_updates = 0;` under `delete from book3 where id = 25;`